utils/src/main/java/messages/ShotMessage.java

```diff
@@ -0,0 +1,25 @@
+package messages;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import model.Shot;
+
+/**
+ * It represents the message containing information about shot.
+ *
+ * @author Emilia Ciastek
+ * @version 1.5
+ */
+public class ShotMessage {
+  private final Shot shot;
+
+  @JsonCreator
+  public ShotMessage(@JsonProperty("shot") Shot shot) {
+    this.shot = shot;
+  }
+
+  @JsonProperty("shot")
```

Comment:
I wonder if that annotation is necessary, if field in JSON body is the same as name of class field and you have proper getter method.